content/user-management copy.md

```diff
@@ -0,0 +1,57 @@
+
+Administrators considered to be the most powerful users in the system. This may be true for some features but not necessarily for all content. In the default structure, administrators have access to everything, but when you build your project and add your custom content, it is perfectly fine if you hide something from the Administrators group. For example you may remove their permissions from confidential documents completely.
+
+By default only the Admin user is a member of this group, but you can add more members as you like. Please use this group for administrative permissions instead of single users.
+
+Editors
+
+We do not have any predefined permissions for this group by default, we consider them as a middle layer between administrators and regular users. For example you can define permissions for the global Editors group for skins and renderers or content types.
+
+
+- Developers: We created this group to help developers work in the development phase of a new project. Group members will see the PRC on the pages, will have access to the Content Explorer. By the default installation the group is the member of the Administrators group.
+
+# Permission
+sensenet offers you a sophisticated permission system, giving you the possibility to apply permission rules according to the logic you prefer.
+Using role-based permissions, you can assign users to roles taking advantage of predefined permission settings. Role based access control provides management of users and tasks that they can perform on contents (for example built-in permissions such as See, Open, Save, Delete ect, and it is also possible to add custom ones as well.
+Define roles based on the business logic right at the beginning of your project so it will be much easier to handle permission related tasks (like add/remove a user) in the long run.
+To learn more go to [role based permission]
+
+# Workspace local groups
+In sensenet the default location of users and groups is the IMS folder. However, we can create groups under workspaces too. The main purpose of Workspace local groups is to help permission management inside workspaces. They are located under the Groups system folder in workspaces and workspace permissions can be defined for these groups. If the local group structure is properly configured, the only thing workspace administrators should do when a new user is assigned to a workspace is to add them to the proper local group(s) instead of assigning permissions to individual users. In sensenet there are three predefined local groups (Owners, Members, Visitors) that cover the most important use cases but there are no limitations to add or remove groups or modify the default behavior. Groups are contents too (like almost everything in sensenet), so you can add additional groups to workspaces as easily as you add folders. Populating a group with users is an easy task as well using our group management GUI with instant search capability.
+
+# How to create a new role
```

Comment:
```suggestion
# How to create a new role

There can be some cases when you need to create a new role, which is actually a new group. You just need to create the group and (as mentioned above), set the proper permission rules of the group, then add the users to the group. This way instead of setting permissions to individual users one by one, they will earn the permissions of the group automaticly. For example, if you would like to create the role "Approval", you create the group "Approval". Then in the subtree you allow the right "approval" and put the users into the group.
```

Reply:
I think we should extend this a bit with more details like:
Groups can be created on the ui(???) and through APIs as well.

automaticly -> automatically